### DISCUSSION TITLE
How to listen keydown and keyup event in EditView

### DISCUSSION THREAD

**Discussion by lisiur** [gyscos/cursive#664] [Q&A] [answered]
I want to listen to the keyboard up and down key events in the EditView to update the selected state of the corresponding SelectView.

I temporarily disabled SelectView and added a global_callback, it would be nice if EditView could provide the ability to add event listeners.

**Comment by gyscos** [comment, marked as answer]
Hi, and thanks for the question!

You can wrap any view into an [`OnEventView`](https://docs.rs/cursive/latest/cursive/views/struct.OnEventView.html), on which you can attach various callbacks.
